test/server/rules.bzl

```diff
@@ -42,3 +42,14 @@ def grakn_test(grakn_artifact = None,
         **kwargs
     )
 
+def grakn_node_test (name = "", feature_label = "@graknlabs_behaviour//"):
+    native.sh_test (
+        name = name,
+        data = [
+            "//:node_modules",
+            "//:package.json",
```

Comment:
I think we generally prefer to omit the colon if we're pointing at a directory? Unsure about this though